recursividade/4-carrinho-compras/carrinhoCompras.js

```diff
@@ -45,20 +45,57 @@ Escolha uma opção: `)
  */
 export async function add(carrinho, produtos) {
   // O código da opção 1 vai aqui
+  console.log(`Digite o numero do item que deseja adicionar ao carrinho:
+  1.Maçã R$0,50/kg
+  2.Pão R$2,20/kg
+  3.Leite R$3,00/Litro
+  4.Frango R$12,00/kg `)
+
+  carrinho.push(produtos[Number(prompt('')) - 1])
```

Comment:
prompt é uma função assíncrona, então precisa ter um await
```suggestion
  const item = await prompt('')
  carrinho.push(produtos[Number(item) - 1])
```